packages/ubuntu_provision/lib/src/eula/eula_page.dart

```diff
@@ -22,12 +42,15 @@ class _EULAPageState extends ConsumerState<EULAPage> {
 
   @override
   Widget build(BuildContext context) {
-    final localeModel = ref.read(localeModelProvider);
     final lang = EULALocalizations.of(context);
-    final eulaFile = File(
-        '/usr/share/desktop-provision/eula/EULA_${localeModel.selectedLocale?.languageCode}.pdf');
-
-    Logger('eula').debug('EULA file: ${eulaFile.path}');
+    final eulaFile = ref.watch(eulaPageProvider).when(
+        data: (eulaFile) => eulaFile.path,
+        loading: () => File('/usr/share/desktop-provision/eula/EULA.pdf').path,
```

Comment:
This will look strange, since it will quickly flip between the files, it's better to have a loader here instead.